From 045dc61b0d5150d47e4a1fe29724344f9e36f10c Mon Sep 17 00:00:00 2001
From: Steven Liu <steven.liu@rock-chips.com>
Date: Sat, 10 Oct 2020 16:15:04 +0800
Subject: [PATCH] arm64: dts: rockchip: modify uart and pwm dtsi for RK3568.

Modify uart and pwm dtsi for RK3568 according to RK3568 TRM.

Signed-off-by: Steven Liu <steven.liu@rock-chips.com>
Change-Id: Ia29916d2c63e1f92c9a6f4e30e0f8d3124895a81
---
 arch/arm64/boot/dts/rockchip/rk3568-pinctrl.dtsi | 6 +++---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi         | 8 ++++----
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568-pinctrl.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-pinctrl.dtsi
index 8366bc52aba7..5c9be2829d38 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-pinctrl.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-pinctrl.dtsi
@@ -1443,7 +1443,7 @@
 	};
 	pwm4 {
 		/omit-if-no-ref/
-		pwm4m4_pins: pwm4m4-pins {
+		pwm4_pins: pwm4-pins {
 			rockchip,pins =
 				/* pwm4 */
 				<0 RK_PC3 1 &pcfg_pull_none>;
@@ -1451,7 +1451,7 @@
 	};
 	pwm5 {
 		/omit-if-no-ref/
-		pwm5m5_pins: pwm5m5-pins {
+		pwm5_pins: pwm5-pins {
 			rockchip,pins =
 				/* pwm5 */
 				<0 RK_PC4 1 &pcfg_pull_none>;
@@ -1459,7 +1459,7 @@
 	};
 	pwm6 {
 		/omit-if-no-ref/
-		pwm6m6_pins: pwm6m6-pins {
+		pwm6_pins: pwm6-pins {
 			rockchip,pins =
 				/* pwm6 */
 				<0 RK_PC5 1 &pcfg_pull_none>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 368c7a7d599a..0dca7d0c058b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -192,7 +192,7 @@
 		reg-io-width = <4>;
 		dmas = <&dmac0 0>, <&dmac0 1>;
 		pinctrl-names = "default";
-		pinctrl-0 = <&uart2m0_xfer>;
+		pinctrl-0 = <&uart0_xfer>;
 		status = "disabled";
 	};
 
@@ -561,7 +561,7 @@
 		reg = <0x0 0xfe6e0000 0x0 0x10>;
 		#pwm-cells = <3>;
 		pinctrl-names = "active";
-		//pinctrl-0 = <&pwm0_pin>;
+		pinctrl-0 = <&pwm4_pins>;
 		clocks = <&cru CLK_PWM1>, <&cru PCLK_PWM1>;
 		clock-names = "pwm", "pclk";
 		status = "disabled";
@@ -572,7 +572,7 @@
 		reg = <0x0 0xfe6e0010 0x0 0x10>;
 		#pwm-cells = <3>;
 		pinctrl-names = "active";
-		//pinctrl-0 = <&pwm0_pin>;
+		pinctrl-0 = <&pwm5_pins>;
 		clocks = <&cru CLK_PWM1>, <&cru PCLK_PWM1>;
 		clock-names = "pwm", "pclk";
 		status = "disabled";
@@ -583,7 +583,7 @@
 		reg = <0x0 0xfe6e0020 0x0 0x10>;
 		#pwm-cells = <3>;
 		pinctrl-names = "active";
-		//pinctrl-0 = <&pwm0_pin>;
+		pinctrl-0 = <&pwm6_pins>;
 		clocks = <&cru CLK_PWM1>, <&cru PCLK_PWM1>;
 		clock-names = "pwm", "pclk";
 		status = "disabled";
-- 
2.35.3

